uptane-standard.md

```diff
@@ -705,9 +704,6 @@ In order to perform full verification, an ECU SHALL perform the following steps:
         1. Check that the non-custom metadata (i.e., length and hashes) of the unencrypted image are the same in both sets of metadata.
         2. Check that the custom metadata (e.g., hardware identifier and release counter) are the same in both sets of metadata.
         3. Check that the release counter in the previous targets metadata file is less than or equal to the release counter in this targets metadata file.
-12. Verify the desired image against its targets metadata.
-  1. If there is no targets metadata about this image, abort the update cycle and report that there is no such image.
-  2. Otherwise, download the image (up to the number of bytes specified in the targets metadata), and verify that its hashes match the targets metadata. (We download up to this number of bytes, because in some cases, the exact number is unknown. This may happen, for example, if an external program is used to compute the root hash of a tree of targets files, and this program does not provide the total size of all of these files.) If consistent snapshots are not used {{metadata_filename_rules}}, then the filename used to download the image file is of the fixed form FILENAME.EXT (e.g., foobar.tar.gz). Otherwise, the filename is of the form HASH.FILENAME.EXT (e.g., c14aeb4ac9f4a8fc0d83d12482b9197452f6adf3eb710e3b1e2b79e8d14cb681.foobar.tar.gz), where HASH is one of the hashes of the targets file listed in the targets metadata file found earlier in step 4. In either case, the client MUST write the file to non-volatile storage as FILENAME.EXT.
```

Comment:
Why is this removed?

Reply:
Yeah, this is pretty important

Reply:
Downloading the image binary and verifying the image against the hashes in the metadata are sections 5.4.3.3. and 5.4.3.4., respectively. This section is explicitly just for *metadata* download and verification.

The rationale for this is that it lets the process be understood as simple, discrete steps:

1. Verify latest attested time (Section 5.4.3.1)
2. Verify metadata (Section 5.4.3.2)
3. Download latest image (Section 5.4.3.3)
4. Verify image (Section 5.4.3.4)
5. Create and send version report (Section 5.4.3.5)

Since metadata verification is so long, involved, and important to Uptane overall, the procedures for partial and full metadata verification are broken out into their own section, and simply referenced from section 5.4.3.2. I could have stuck both procedures as sub-sections of 5.4.3.2 instead, but in my editorial judgement it doesn't flow well that way.

Reply:
Got it. Where do I preview a rendering of this file, please?

Reply:
https://uptane.github.io/uptane-standard/PR-20/uptane-standard.html